src/objects/DeploymentEvent__e.object

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm new to platform events so forgive me if this is naive BUT, I think we want to be careful about how we manage this shared resource for looking to the future. To that end, 
1. Do we need to name this something that makes it clearly bdi-specific, looking forward to a time when other teams and products will probably be firing similar deployment events? Or is this event generic to those use cases? AND
2. Do we need to do something to avoid spamming other listeners, especially if this IS an event that's shared for other products, we would presumably need to avoid triggering each others listeners. Or is the DeploymentId__c enough to tie it back to the specific deployment that launched it? 
@kenlewis-sfdo thoughts?

Reply:
Good questions!  I'm totally open to discussing this further - my thought was to keep it as generic as possible, for exactly that purpose, so that it can be reused in various other parts of the application without looking like it "belongs" to one area of functionality or another.  It is quite clearly tied to a deployment event, based on its custom fields, but not tied to any specifically deployed content (type of metadata)...

Also, listeners have to first subscribe to the event's stream, then can do what they want with each event received.  The idea of checking the deployment Id and verifying it against our registered Ids was for exactly that purpose, so that we can be sure to only respond to ours and not any one else's DeploymentEvent platform events, in the future possible case that this is used elsewhere.  Not sure though of the pros/cons of re-using one type of Platform Event vs. creating more specific events for each application/purpose...

Reply:
This seems great and I think this approach does fine now that I understand it! Thanks!